cg/store/filters/status_order_filters.py

```diff
@@ -6,27 +6,31 @@
 from cg.store.models import Order
 
 
-def filter_orders_by_workflow(orders: Query, workflow: str, **kwargs) -> Query:
-    """Return orders filtered on workflow."""
+def filter_orders_by_workflow(orders: Query, workflow: str | None, **kwargs) -> Query:
```

Comment:
I like the removal of the docstrings which just state what the method is named 💪